source/rules/roles.txt

```diff
@@ -13,22 +13,27 @@ Role-based Permissions
    :depth: 2
    :class: singlecol
 
-.. include:: /includes/non-sync-rules-note.rst
-
 Overview
 --------
 
-You secure your App's data by defining roles that are automatically
-assigned to incoming user requests. Each role has fine-grained data
-access permissions and dynamic conditions that determine when the role
+You secure your App's data by defining **roles** that are automatically assigned
+to incoming user requests or Device Sync sessions. Each role has fine-grained
+data access permissions and dynamic conditions that determine when the role
 applies.
 
+For examples of how you might configure permissions for common scenarios with
+Device Sync, see :ref:`flexible-sync-permissions-guide`.
```

Comment:
[minor] Add definite article because the permissions guide is a known entity.
```suggestion
Device Sync, see the :ref:`flexible-sync-permissions-guide`.
```